appendix/single-sign-on.rst

```diff
@@ -280,50 +287,38 @@ how to reach the *domain controller* (Active Directory server).
                                     (must not be read-only,
                                     but can be the same as ``<domain-controller>``)
 
-.. tabs::
+   .. code-block::
```

Comment:
In my opinion the code block could be moved out of the note box. But I'm fine with that, too.